packages/react-dropdown/src/MultiDownshift.js

```diff
@@ -41,14 +41,10 @@ type State = {
 };
 
 class MultiDownshift extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-
-    this.state = {
-      selectedItems: this.props.defaultSelectedItems,
-      inputValue: this.getDefaultInputValue() || '',
```

Comment:
this function never returns null/undefined, at worst it will return an empty string. So this `|| ''` is unneeded